### ISSUE TITLE
mark tests as pending

### ISSUE THREAD

**Issue by yoshuawuyts** [yoshuawuyts/cuke-tap#5]
Sometimes tests are not ready yet. For that there's the `@todo` annotation:

``` gherkin
@todo
Feature: myCoolFeature
  bla bla bla
```

It'd be nice if we could translate that under the hood to [test.skip()](https://github.com/substack/tape#testskipname-cb) - by doing that we'd be capable of adding features to the repo in bulk, and crossing them off one-by-one. Also nice to view progress for example.

Could then even go further and look at git commits between the feature added, and the feature passing to determine what was the root cause of a certain code change. :rainbow: endless possibilities :rainbow: 

Anyway, this might be useful. Thoughts?

**Comment by ahdinosaur** [comment]
yep, keen for using a `@todo` annotation to skip the test. :smile:
